relayer/processor/types_internal.go

```diff
@@ -196,6 +198,38 @@ type outgoingMessages struct {
 	chanMsgs []channelMessageToTrack
 }
 
+// MarshalLogObject satisfies the zapcore.ObjectMarshaler interface
+// so that you can use zap.Object("messages", r) when logging.
+// This is typically useful when logging details about a partially sent result.
+func (om *outgoingMessages) MarshalLogObject(enc zapcore.ObjectEncoder) error {
+	for i, m := range om.pktMsgs {
+		pfx := "pkt_" + strconv.FormatInt(int64(i), 10) + "_"
+		enc.AddString(pfx+"evt_type", m.msg.eventType)
```

Comment:
Should these read `event_type`?

Reply:
I tried to keep the log keys as slim as possible, but happy to expand for clarity